graphics/vector3_for_2d_rendering.py

```diff
@@ -0,0 +1,70 @@
+"""
+render 3d points for 2d surfaces.
+"""
+
+import math
+
+__version__ = "2020.9.26"
+__author__ = "xcodz-dot"
+
+
+def convert_to_2d(x: float, y: float, z: float, scale: float,
+                  distance: float) -> (float, float):
+    """
+    Converts 3d point to a 2d drawable point
+
+    >>> convert_to_2d(1.0, 2.0, 3.0, 10.0, 10.0)
+    (7.6923076923076925, 15.384615384615385)
+    """
+    if not isinstance(x, (float, int)):
+        raise ValueError("x must be float or int")
+    if not isinstance(y, (float, int)):
+        raise ValueError("y must be float or int")
+    if not isinstance(z, (float, int)):
+        raise ValueError("z must be float or int")
+    if not isinstance(scale, (float, int)):
+        raise ValueError("scale must be float or int")
+    if not isinstance(distance, (float, int)):
+        raise ValueError("distance must be float")
```

Comment:
Can we compress all these expressions into a single one like so?

```python
if not all(isinstance(val, (float, int)) for val in locals().values()):
    raise ValueError(f"Input values must either be float or int: {list(locals().values())}")
```

Reply:
yes, i will be changing

Reply:
done